lib/server-jobs.js

```diff
@@ -393,6 +393,8 @@ module.exports.errorAbandonedJobs = function (callback) {
             }
             callback(null);
           });
+        } else {
+          callback(null);
```

Comment:
This was causing test failures, and looks to just be a very longstanding bug!

Reply:
Oh wow, good find!